nion/swift/ExportDialog.py

```diff
@@ -241,9 +241,14 @@ def __init__(self, document_controller: DocumentController.DocumentController, d
 
         def ok_clicked() -> bool:
             dpi = 96
-            width_px = (handler.width_model.value or display_size.width) * dpi
-            height_px = (handler.height_model.value or display_size.height) * dpi
+            if display_item.display_data_shape is not None:
+                display_item_ratio = display_item.display_data_shape[1] / display_item.display_data_shape[0]
+            else:
+                display_item_ratio = 1
+
```

Comment:
This is an extra blank line - to keep the source code easy to read, only one blank line needs to appear between code sections. This code should go away, but it's good to have clean white space changes, if any.